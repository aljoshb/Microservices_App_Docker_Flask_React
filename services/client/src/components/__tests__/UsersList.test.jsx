import React from 'react';
import { shallow } from 'enzyme';

import UsersList from '../UsersList';

const users = [
    {
        'active': true,
        'email': 'bolualawode@gmail.com',
        'id': 1,
        'username': 'josh'
    },
    {
        'active': true,
        'email': 'jon@doe.com',
        'id': 2,
        'username': 'jondoe'
    }
];

test('UsersList renders properly', () => {
    const wrapper = shallow(<UsersList users={users}/>);
    const element = wrapper.find('h4');
    expect(element.length).toBe(2);
    expect(element.get(0).props.children).toBe('josh')
})