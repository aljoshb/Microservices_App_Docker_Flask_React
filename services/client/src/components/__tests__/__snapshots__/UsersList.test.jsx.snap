// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersList renders a snapshot properly 1`] = `
<div>
  <h4
    className="box title is-4"
  >
    josh
  </h4>
  <h4
    className="box title is-4"
  >
    jondoe
  </h4>
</div>
`;
